### ISSUE TITLE
'calls' command should also (optionally) display the phone number

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#310]
```
It currently looks something like this

Jan 2, 2013 3:50:25 PM - Foo Bar - Outgoing of 3s
Jan 2, 2013 4:35:03 PM - Snafu Smith - Outgoing of 1min 56s

but I often also want to know the origin/called number. So change it to

Jan 2, 2013 3:50:25 PM - Foo Bar (+4988881111)- Outgoing of 3s
Jan 2, 2013 4:35:03 PM - Snafu Smith (+122233333) - Outgoing of 1min 56s
```

Original issue reported on code.google.com by `fschm...@gmail.com` on 3 Jan 2013 at 3:08

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 26 Jan 2013 at 5:37

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 26 Jan 2013 at 6:56
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
http://code.google.com/p/gtalksms/source/detail?r=094b5da3d487b985464a3b302cbb72
acb50d83af
```

Original comment by `fschm...@gmail.com` on 26 Jan 2013 at 7:07

**Comment by GoogleCodeExporter** [comment]
```
New version 4.4 Beta 1 is available in the download section
```

Original comment by `Florent....@gmail.com` on 16 Feb 2013 at 9:47

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 4.4
```

Original comment by `Florent....@gmail.com` on 15 Sep 2013 at 3:27
- Changed state: **Deployed**
